app/views/contexts/summary/question.py

```diff
@@ -39,7 +39,16 @@ def __init__(
         )
 
     def _get_answer(self, answer_store, answer_id):
-        return answer_store.get_escaped_answer_value(answer_id, self.list_item_id)
+        if answer_store.get_escaped_answer_value(
```

Comment:
This doesn't need to call answer_store.get_escaped_answer_value(answer_id, self.list_item_id) 3 times. Just assign it a var at the start

Reply:
Ok, removed all repetition.